plugins/gatsby-plugin-generate-doc-json/utils/handlers.js

```diff
@@ -0,0 +1,151 @@
+const all = require('mdast-util-to-hast/lib/all');
+const { findAttribute } = require('../../../codemods/utils/mdxast');
+const toString = require('mdast-util-to-string');
+const u = require('unist-builder');
+
+const stripNulls = (obj) =>
+  Object.fromEntries(Object.entries(obj).filter(([, value]) => value != null));
+
+const getAllAttributes = (node) =>
+  node.attributes
+    ? node.attributes.reduce((obj, { name: propName, value }) => {
+        return { ...obj, [propName]: value };
+      }, {})
+    : {};
+
+module.exports = {
+  image: (h, node, imageHashMap, fileRelativePath) => {
+    const srcUrl = fileRelativePath
+      .replace('.mdx', '')
```

Comment:
this assumes that the relativePath will always be under `./images/` for the file image